tests/plugin-parser/unittest_mlops_plugin_parser.cc

```diff
@@ -381,49 +469,120 @@ TEST_F (PkgMgrInfoMockTestFixture, test_p1)
   EXPECT_CALL(*pkgMgrInfoMockInstance, pkgmgrinfo_pkginfo_get_pkginfo (_, _))
       .WillOnce (Return (PMINFO_R_OK));
 
+  char pkg_type_tpk[] = "tpk";
+  EXPECT_CALL(*pkgMgrInfoMockInstance, pkgmgrinfo_pkginfo_get_type (_, _))
+      .WillOnce (DoAll (SetArgPointee<1> (pkg_type_tpk),
+                        Return (PMINFO_R_OK)));
+
+  EXPECT_EQ (exec_plugin_parser_func ("PKGMGR_MDPARSER_PLUGIN_INSTALL", pkgid, appid, NULL), 0);
+}
+
+/**
+ * @brief Positive test case of sample rpk package install/upgrade/uninstall.
+ */
+TEST_F (PkgMgrInfoMockTestFixture, rpk_scenario_p1)
+{
+  std::string pkgid = "pkgid";
+  std::string appid = "appid";
+
+  EXPECT_CALL (*pkgMgrInfoMockInstance, pkgmgrinfo_pkginfo_destroy_pkginfo (_))
+      .WillRepeatedly (Return (PMINFO_R_OK));
+
+  EXPECT_CALL(*pkgMgrInfoMockInstance, pkgmgrinfo_pkginfo_get_pkginfo (_, _))
+      .WillRepeatedly (Return (PMINFO_R_OK));
+
   char pkg_type_rpk[] = "rpk";
   EXPECT_CALL(*pkgMgrInfoMockInstance, pkgmgrinfo_pkginfo_get_type (_, _))
-      .WillOnce (DoAll (SetArgPointee<1> (pkg_type_rpk),
+      .WillRepeatedly (DoAll (SetArgPointee<1> (pkg_type_rpk),
                         Return (PMINFO_R_OK)));
 
   char root_path[] = "../tests/plugin-parser/test_rpk_samples/working_rpk";
   EXPECT_CALL(*pkgMgrInfoMockInstance, pkgmgrinfo_pkginfo_get_root_path (_, _))
-      .WillOnce (DoAll (SetArgPointee<1> (root_path),
+      .WillRepeatedly (DoAll (SetArgPointee<1> (root_path),
                         Return (PMINFO_R_OK)));
 
   char res_type[] = "sample-res-type";
   EXPECT_CALL(*pkgMgrInfoMockInstance, pkgmgrinfo_pkginfo_get_res_type (_, _))
-      .WillOnce (DoAll (SetArgPointee<1> (res_type),
+      .WillRepeatedly (DoAll (SetArgPointee<1> (res_type),
                         Return (PMINFO_R_OK)));
 
   char res_version[] = "1.5.0";
   EXPECT_CALL(*pkgMgrInfoMockInstance, pkgmgrinfo_pkginfo_get_res_version (_, _))
-      .WillOnce (DoAll (SetArgPointee<1> (res_version),
+      .WillRepeatedly (DoAll (SetArgPointee<1> (res_version),
                         Return (PMINFO_R_OK)));
 
+  // install rpk
```

Comment:
can out update all line-comment in test? (local rule for line comment : // > /* */)

Reply:
My mistake, thanks! I updated those lines